src/PKSim.Presentation/Services/CompoundAlternativeTask.cs

```diff
@@ -241,7 +241,7 @@ public ICommand EditSolubilityTableFor(IParameter parameter)
             if (!tablePresenter.Edit(parameter))
                return new PKSimEmptyCommand();
 
-            return _parameterTask.SetParameterFomula(parameter, tablePresenter.EditedFormula);
+            return _parameterTask.UpdateTableFormula(parameter, tablePresenter.EditedFormula);
```

Comment:
This is not required here but much consistent with the way it should be used anyways. Instead of SETTING the formula, I am updating the formula with the edited one